### PR TITLE
Update tests to use object builders in pkg/builder

### PR DESCRIPTION
**Pull request by skriss**
@carlisia @nrb @prydonius I'm not sure what the best approach is for reviewing this. Possibilities:

- I split it up into 10-20 smaller PRs, each one covering a separate object
- I leave it as-is and you sample/scan
- We take the fact that all unit tests are passing as an indication that the updates were correct, and merge as-is

Thoughts?

### CODE REVIEW

**Comment by skriss** [comment]
I'd be comfortable with a light review/relying on the tests passing, as this was a lot of find/replace and some manual fixes.

**Comment by carlisia** [comment]
I'm comfortable with a light review. Doing that now.

File: pkg/controller/schedule_controller_test.go [hunk 1]
```diff
@@ -259,41 +262,41 @@ func parseTime(timeString string) time.Time {
 func TestGetNextRunTime(t *testing.T) {
```
Comment (nrb) [inline pkg/controller/schedule_controller_test.go]: It looks like all the remaining tests in this file use the struct literals.

**Comment by nrb** [comment]
Everything looks good except for some literals in `schedule_controller_test.go`.

File: pkg/controller/schedule_controller_test.go (hunk 1 above)
Comment (skriss) [reply pkg/controller/schedule_controller_test.go]: Got those updated

**Comment by skriss** [comment]
OK, took it out of draft status, ready for merge!

**Pull request merged**